### ISSUE TITLE
srvsize not updated after image deleted from disk

### ISSUE THREAD

**Issue by abulhol** [FOGProject/fogproject#390]
When I manually delete an image from the /images folder (rm -r /images/imageXY), the FOG web app still thinks that it is there (I delete the image files this way due to data privacy requirements). 
I.e. the web API reports the old `srvsize`. 
Related: https://github.com/FOGProject/fogproject/issues/384
I have tried `sudo systemctl restart FOGImageSize`, but it does not pick up the new information. 
What can I do to make the web DB update this column?

**Comment by Sebastian-Roth** [comment]
@abulhol The service does not update the size unless it find's a directory. Something we can change in the code. But you can manually fix this for now via:
```
sudo -i
mkdir /images/imageXY
systemctl restart FOGImageSize
```

**Comment by abulhol** [comment]
Thanks! I was deleting the directory itself as well, now I am just deleting the files inside, so it works.

**Comment by Sebastian-Roth** [comment]
@abulhol I though about keeping this open as we could actually change the code to set srvsize to zero if it cannot find the directory at all.

**Comment by abulhol** [comment]
agree, good idea

**Comment by Sebastian-Roth** [comment]
@abulhol Oh well, lost track of this. But finally pushed 7e12c2b39 to fix this.

**Comment by Sebastian-Roth** [comment]
Also fixed in `working-1.6`: e1ce5559f
